daemon/debugtrap_windows.go

```diff
@@ -0,0 +1,30 @@
+package daemon
```

Comment:
I would probably add the windows build tag to this file to be explicit and that is what we do for most files in the codebase.